src/main/java/uk/gov/hmcts/darts/event/client/DarNotifyEventClient.java

```diff
@@ -97,4 +108,41 @@ private static OffsetDateTime dateTimeFrom(Event event) {
         );
     }
 
+    static class DarPcTimeLogInterceptor implements ClientInterceptor {
+
+        @Override
+        public boolean handleRequest(MessageContext messageContext) {
+            return true;
+        }
+
+        @Override
+        public boolean handleResponse(MessageContext messageContext) {
+            return true;
+        }
+
+        @Override
+        public boolean handleFault(MessageContext messageContext) {
+            return true;
+        }
+
+        @Override
+        public void afterCompletion(MessageContext messageContext, Exception ex) {
+            try {
+                HttpComponentsConnection connection = (HttpComponentsConnection) TransportContextHolder.getTransportContext().getConnection();
+
+                Header dateHeader = connection.getHttpResponse().getFirstHeader("Date");
+                OffsetDateTime responseDateTime = OffsetDateTime.parse(dateHeader.getValue(), DateTimeFormatter.RFC_1123_DATE_TIME);
+                OffsetDateTime currentTime = OffsetDateTime.now();
```

Comment:
Can we make this an injectable component, such that we can more easily mock the "current time" in tests?